shopfloor_base/models/shopfloor_app.py

```diff
@@ -86,6 +86,17 @@ class ShopfloorApp(models.Model):
         help="If set, the app will be first loaded with this lang.",
     )
     lang_ids = fields.Many2many("res.lang", string="Available languages")
+    storage_driver = fields.Selection(
```

Comment:
this is very low level stuff, pls drop it for now

Reply:
Dropped.